docs/wiki/EL/evm.md

```diff
@@ -0,0 +1,623 @@
+# The Ethereum Virtual Machine (EVM)
+
+You have likely observed that Ethereum transactions take some time to finalise, [around 12 seconds](https://web.archive.org/web/20240304145956/https://etherscan.io/chart/blocktime) at the time of writing. A series of crucial steps occur during this timeframe. The transaction is first queued in a pool. Then, it is selected by a node and executed by a special program - **the Ethereum Virtual Machine (EVM)**, the result of which is stored on the blockchain.
+
+This article describes the role of EVM in Ethereum ecosystem and how it works. As the EVM executes transactions, it modifies the overall state of Ethereum. In that regard, Ethereum can be modeled as a **state machine**.
+
+## The Ethereum state machine
+
+In computer science, **state machine** is an abstraction used to model the behavior of a system. It illustrates how a system can be represented by a set of distinct states and how inputs can drive changes in the state.
+
+A familiar is a vending machine, an automated system for dispensing products upon receiving payment.
```

Comment:
I guess you meant 'A familiar examples.."?

Reply:
Yep 😅